config/samples/datadog-agent-all.yaml

```diff
@@ -3,19 +3,41 @@ kind: DatadogAgent
 metadata:
   name: datadog
 spec:
+  clusterName: <CLUSTER_NAME>
   credentials:
     apiKey: <DATADOG_API_KEY>
     appKey: <DATADOG_APP_KEY>
+    token: <DATADOG_CLUSTER_AGENT_TOKEN>
   agent:
     image:
       name: "datadog/agent:latest"
```

Comment:
Not sure we should include `latest` here, we should be in line with Helm chart and use fixed version. In terms of example we should maybe remove this option to let the operator choose

Reply:
currently the operator doesn't have that logic yet (to default image.name). I guess we'll have to put a version here